Crypto101.bib

```diff
@@ -282,3 +282,42 @@ @misc{schneier:ipsec
   year = {1999},
   note = {\url{https://www.schneier.com/paper-ipsec.pdf}}
 }
+@misc{cryptoeprint:2004:199,
+  author = {Xiaoyun Wang and Dengguo Feng and Xuejia Lai and Hongbo Yu},
+  title = {Collisions for Hash Functions MD4, MD5, HAVAL-128 and RIPEMD},
+  howpublished = {Cryptology ePrint Archive, Report 2004/199},
+  year = {2004},
+  note = {\url{http://eprint.iacr.org/2004/199}}
+}
+@INPROCEEDINGS{article143,
+  author =    {Bert den Boer and
+               Antoon Bosselaers},
+  title =     {{Collisions for the compression function of MD5}},
+  year =      {1993},
+  pages =     {293-304},
+  booktitle = {Advances in Cryptology - EUROCRYPT 1993},
+  editor =    {Tor Helleseth},
+  volume =    {765},
+  number =    {},
+  series =    {Lecture Notes in Computer Science},
+  address =   {Lofthus,N},
+  publisher = {Springer-Verlag}
```

Comment:
Consider adding a PDF link to the paper (e.g. https://www.cosic.esat.kuleuven.be/publications/article-143.pdf) so that people that haven't given Springer a bunch of money can also go read it :)